nix/docker/context/bin/functions

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+#####################################################################
+#
+# Trap the SIGTERM signal
+#
+trapSIGTERM () {
+
+  PID="$!"
+  trap "sigtermHandler $PID" SIGTERM
```

Comment:
Why not just set `STOPSIGNAL` in the docker image, or 
```
config = {
      EntryPoint = [ "${entry-point}/bin/entry-point" ];
      StopSignal = "INT";
    };
```
if using nix?

Reply:
I looked into this and it would work. However, if we do this we loose

```
Signalling for shutdown ...
```

which comes from the wrapper script that runs cardano-node as a sub-process in the background. I personally would prefer to make this change dependent on 

* [#2648] Indicate graceful shutdown with a log message

Here is the issues in the incubator project, where we can follow up on this: 

* [#49](https://github.com/tdiesler/nessus-cardano/issues/49) Simplify stop signal handling by using SIGINT directly